unstructured_inference/visualize.py

```diff
@@ -1,21 +1,23 @@
 # Copyright (c) Megvii Inc. All rights reserved.
 # Unstructured modified the original source code found at
 # https://github.com/Megvii-BaseDetection/YOLOX/blob/ac379df3c97d1835ebd319afad0c031c36d03f36/yolox/utils/visualize.py
+from typing import Optional, Union
 
 import cv2
+import matplotlib.pyplot as plt
```

Comment:
`matplotlib` is already in our dependencies via `pycocotools`, but to be safe we should add it explicitly to `requirements/base.in` if we're going to import it directly. After adding it, run `make pip-compile` to recompile dependencies.

Reply:
Sounds good!